index.js

```diff
@@ -6,10 +6,7 @@ import { zpad, str2hex } from './helpers.js'
 
 /**
  * AMP Protocol documentation:
- * http://www.gvgdevelopers.com/concrete/apis/amp_protocol/
- * http://www.gvgdevelopers.com/Protocols/AMP_SDK/Docs/AMP%20at%20a%20glance.pdf
```

Comment:
These are also available via the web archive if they might provide more info:
https://web.archive.org/web/20231210200954/http://www.gvgdevelopers.com/Protocols/AMP_SDK/Docs/AMP%20at%20a%20glance.pdf